### DISCUSSION TITLE
Template Database help

### DISCUSSION THREAD

**Discussion by ghost** [Open-Systems-Pharmacology/Forum#822] [Questions & Problems]
I new to PK-Sim and still learning; I am currently working on the tutorial "PK Drug-Drug-Interaction: Simulating the DDI of rifampicin and midazolam"; I am unable to open any of the EX_DDI_RIF files since my version of PK-Sim is new and not compatible with those files. However, I am able to load from templates for the rifampicin and completed that portion. My issue is that when I try to load administration protocols, under predefined templates I only have a 25mg warfarin sodium protocol and rifampicin 600mg protocol and I do not have anything else listed. Same with observed data, I only have the Baneyx 2013 rifampin and Peloquin rifampin. Did I download incorrectly and I am missing these templates?

**Comment by msevestre** [comment]
@nsutlian 
Can you clarify
> I am unable to open any of the EX_DDI_RIF files since my version of PK-Sim is new 

PK-Sim is backward compatible and can open project starting with  version 6.0 which was released like 5 years ago+

Is it possible that the tutorials that you are using is asking you to use a specific template database ?

**Reply by PauAce** [reply]
I can confirm I have the same issue trying to open said file, as I am running through the same training.  Seems that the training was created in a rather early version of the software. If there is a solution to this that doesn't involve installing an old version of PK Sim that would be great, as I am also trying to complete this training. Thanks

![image](https://user-images.githubusercontent.com/29316067/184720125-396f1a5c-dcdd-49b0-bf87-d87126ad1db6.png)

**Reply by Yuri05** [reply]
You can open the project with an older version of PK-Sim portable version.
Portable versions does not require any installation (just unzip the archive) and can be used in parallel to the existing OSP Suite installation.

E.g. try it with [PK-Sim 8.0 portable](https://github.com/Open-Systems-Pharmacology/PK-Sim/releases/download/v8.0/pk-sim-portable-setup.zip). 
Once you opened the project, you could either work with it in PK-Sim 8. Or just save it once and reopen in the current PK-Sim version.

**Comment by StephanSchaller** [comment]
@nsutlian , @PauAce , if you are looking for up to date training material (all exercises are prepared for either V9, V10 or V11 (not all exercises have yet been translated to V11)), please check out our online training platform for the OSP Suite:

This includes exercises for DDI

Access for Academia: https://learn-esqlabs.talentlms.com/index
Access for Industry:  https://training-esqlabs.talentlms.com/

FYI @PavelBal,  @maalbrec89
